### DISCUSSION TITLE
Optimistic locking

### DISCUSSION THREAD

**Discussion by acbeni** [aarondl/sqlboiler#913] [Q&A] [answered]
This might be obvious to solve, but i m kinda new in sqlboiler and didn't find much pointers about optimistic locking.

Does sqlboiler support any methods to ease the implementation of optimistic locking?

Thanks in advance

**Comment by aarondl** [comment, marked as answer]
It doesn't do anything for you. You've got to do it manually. Note `qm.Lock` to add lock statements on to the ends of queries :)

**Reply by acbeni** [reply]
Thanks  @aarondl  for your reply.
and thank you very much for this wonderful framework ❤️
